Cesium.Parser.Tests/ParserTests/FullParserTests.cs

```diff
@@ -67,4 +67,16 @@ int main()
     int f = foo();
     return foo() + 1;
 }");
+
+    [Fact]
+    public Task StringLiteral() => DoTest(@"int main()
+{
+    test(""hello world"");
+}");
+
+    [Fact]
+    public Task StringVariable() => DoTest(@"int main()
```

Comment:
Does current implementation handle case of non-const `char*`?

Reply:
As of now, `const` is read and then almost completely ignored by the compiler.

(the only exception is that we'll throw an error on `const const` declaration; that's it)

Also, I've tried reading the standard yesterday, but failed to figure out if it's even allowed to have a non-`const` string literal. Is it?